tests/11_reactions/ref_output/flow_decay_long_gmsh/mass_balance.txt

```diff
@@ -23,483 +23,483 @@
-9.5	"ALL"	"C"	-1.53573e-15	3.73035e-16	-1.90876e-15	4.83543	0	0	0	-7.65199e-16	0.0135088	-9.27886e-15	4.43043	-7.10543e-15
-9.5	"ALL"	"D"	2.13153e-16	2.13163e-16	-1.01676e-20	2.57574e-05	0	0	0	1.06572e-16	-2.31228e-05	1.54544e-15	-0.539974	-3.24922e-18
-9.5	"ALL"	"E"	2.66453e-16	2.66454e-16	-5.21396e-22	1.32084e-06	0	0	0	1.33226e-16	-1.31728e-06	2.2645e-15	-0.674999	-3.12176e-16
-9.5	"ALL"	"F"	2.84121e-16	2.84217e-16	-9.65703e-20	0.000244639	0	0	0	1.42022e-16	-0.000195166	1.89973e-15	-0.809755	-3.20653e-16
-9.5	"ALL"	"G"	2.94954e-16	3.01981e-16	-7.02638e-18	0.0177998	0	0	0	1.45658e-16	-0.00921785	1.11992e-15	-0.9272	-4.30211e-16
-9.5	"ALL"	"H"	3.18252e-16	3.19744e-16	-1.49175e-18	0.00377902	0	0	0	1.58675e-16	-0.00228441	2.06958e-15	-1.07622	-1.37911e-16
-10	"region_1"	"A"	0	0	0	0.000776133	0	0	0	0	0	0	0	0
-10	"region_1"	"B"	0	0	0	0.000854711	0	0	0	0	0	0	0	0
-10	"region_1"	"C"	0	0	0	4.84433	0	0	0	0	0	0	0	0
-10	"region_1"	"D"	0	0	0	1.35382e-05	0	0	0	0	0	0	0	0
-10	"region_1"	"E"	0	0	0	6.6131e-07	0	0	0	0	0	0	0	0
-10	"region_1"	"F"	0	0	0	0.000135363	0	0	0	0	0	0	0	0
-10	"region_1"	"G"	0	0	0	0.0115655	0	0	0	0	0	0	0	0
-10	"region_1"	"H"	0	0	0	0.00232904	0	0	0	0	0	0	0	0
-10	"IMPLICIT BOUNDARY"	"A"	3.54965e-16	3.55271e-16	-3.06375e-19	0	0	0	0	0	0	0	0	0
-10	"IMPLICIT BOUNDARY"	"B"	2.12825e-16	2.13163e-16	-3.37394e-19	0	0	0	0	0	0	0	0	0
-10	"IMPLICIT BOUNDARY"	"C"	-1.53924e-15	3.73035e-16	-1.91228e-15	0	0	0	0	0	0	0	0	0
-10	"IMPLICIT BOUNDARY"	"D"	2.13157e-16	2.13163e-16	-5.34416e-21	0	0	0	0	0	0	0	0	0
-10	"IMPLICIT BOUNDARY"	"E"	2.66453e-16	2.66454e-16	-2.6105e-22	0	0	0	0	0	0	0	0	0
-10	"IMPLICIT BOUNDARY"	"F"	2.84164e-16	2.84217e-16	-5.3434e-20	0	0	0	0	0	0	0	0	0
-10	"IMPLICIT BOUNDARY"	"G"	2.97415e-16	3.01981e-16	-4.56544e-18	0	0	0	0	0	0	0	0	0
-10	"IMPLICIT BOUNDARY"	"H"	3.18825e-16	3.19744e-16	-9.1938e-19	0	0	0	0	0	0	0	0	0
-10	"ALL"	"A"	3.54965e-16	3.55271e-16	-3.06375e-19	0.000776133	0	0	0	1.77387e-16	-0.00048311	3.31862e-15	-0.134224	-1.28261e-16
-10	"ALL"	"B"	2.12825e-16	2.13163e-16	-3.37394e-19	0.000854711	0	0	0	1.06292e-16	-0.000610402	1.22608e-15	-0.269145	-3.79145e-16
-10	"ALL"	"C"	-1.53924e-15	3.73035e-16	-1.91228e-15	4.84433	0	0	0	-7.67865e-16	0.00889988	-1.00467e-14	4.43933	-7.10543e-15
-10	"ALL"	"D"	2.13157e-16	2.13163e-16	-5.34416e-21	1.35382e-05	0	0	0	1.06576e-16	-1.22192e-05	1.65202e-15	-0.539986	3.57482e-17
-10	"ALL"	"E"	2.66453e-16	2.66454e-16	-2.6105e-22	6.6131e-07	0	0	0	1.33227e-16	-6.5953e-07	2.39772e-15	-0.674999	-3.48029e-16
-10	"ALL"	"F"	2.84164e-16	2.84217e-16	-5.3434e-20	0.000135363	0	0	0	1.4206e-16	-0.000109276	2.04179e-15	-0.809865	-2.81161e-16
-10	"ALL"	"G"	2.97415e-16	3.01981e-16	-4.56544e-18	0.0115655	0	0	0	1.47477e-16	-0.00623424	1.2674e-15	-0.933434	-4.7011e-16
-10	"ALL"	"H"	3.18825e-16	3.19744e-16	-9.1938e-19	0.00232904	0	0	0	1.59126e-16	-0.00144997	2.22871e-15	-1.07767	-1.86049e-16
+0.5	"region_1"	"A"	0	0	0	0.109221	0	0	0	0	0	0	0	0
+0.5	"region_1"	"B"	0	0	0	0.499034	0	0	0	0	0	0	0	0
```

Comment:
Indeed, the new reference results are very precise (compared to refined time step).

Plus/minus signs at line beginnings come from diff.